kiosk/kiosk_browser/browser_widget.py

```diff
@@ -1,21 +1,31 @@
 import re
+import urllib
+import logging
 from PyQt5.QtCore import QTimer
 from PyQt5.QtWidgets import QShortcut
-from PyQt5.QtWebEngineWidgets import QWebEngineView, QWebEnginePage, QWebEngineProfile
+from PyQt5.QtWebEngineWidgets import QWebEngineView
 from PyQt5.QtWidgets import QSizePolicy
 
 from kiosk_browser import system
 
 class BrowserWidget(QWebEngineView):
 
-    def __init__(self, url, *args, **kwargs):
+    def __init__(self, url, get_current_proxy, *args, **kwargs):
         QWebEngineView.__init__(self, *args, **kwargs)
 
+        # Authenticate proxies
```

Comment:
Nitpick: Update comment to clarify that this merely registers the handler for authentication, should it be required.

Reply:
https://github.com/dividat/playos/pull/54/commits/811d5c06647486393f4cdf76aae3a041fecfb4ff